app/assets/v2/js/grants/index.js

```diff
@@ -121,8 +137,8 @@ Vue.component('grant-sidebar', {
             target = `/grants/clr/${this.customer_name}/${this.round_num}/${this.sub_round_slug}?type=${params.type}`;
           }
         }
-
-        document.location.href = target;
+        fetchGrants(this.page)
+        // document.location.href = target;
```

Comment:
do we wanna keep this commented out ?

Reply:
This whole block is unreachable (following line #113) is this deliberate?